lib/galaxy/objectstore/__init__.py

```diff
@@ -662,7 +662,7 @@ def __get_store_id_for(self, obj, **kwargs):
             # if this instance has been switched from a non-distributed to a
             # distributed object store, or if the object's store id is invalid,
             # try to locate the object
-            log.warning('The backend object store ID (%s) for %s object with ID %s is invalid'
+            log.debug('The backend object store ID (%s) for %s object with ID %s is invalid'
```

Comment:
Could we redo the log to be more precise in the warnings ?
I guess you're hitting this with old files that were created prior to switching to a distributed object store, which isn't warning-worthy. But this `else` also captures the case of a potentially invalid object store id, which is probably something an admin should look into. What do you think about https://gist.github.com/mvdbeek/f1027cba2ed3923202e6c5de8a9a9466

Reply:
(I'm not terribly familiar with the objectstore code in the first place.)

yeah that change looks fine. I just really don't want a noisy warning emitted unless it's necessary and the admin can actually do something to rectify it.

Reply:
I'll apply it to this branch